e2e/build.gradle

```diff
@@ -0,0 +1,40 @@
+subprojects {
+    buildscript {
+        repositories {
+            maven {
+                url "https://plugins.gradle.org/m2/"
+            }
+        }
+        dependencies {
+            classpath 'com.bmuschko:gradle-docker-plugin:7.0.0'
+        }
+    }
+
+    sourceSets {
+        integrationTest {
+            java {
+                compileClasspath += main.output + test.output
+                runtimeClasspath += main.output + test.output
+                srcDir file('src/integrationTest/java')
+            }
+            resources.srcDir file('src/integrationTest/resources')
+        }
+    }
+
+    configurations {
+        integrationTestImplementation.extendsFrom testImplementation
+        integrationTestRuntime.extendsFrom testRuntime
+    }
+
+    def DATA_PREPPER_CORE_JAR = project(':data-prepper-core').jar
+
+    task copyDataPrepperJar(type: Copy) {
+        dependsOn DATA_PREPPER_CORE_JAR
+        from("${project(':data-prepper-core').jar.archivePath}")
+        into 'build/bin'
+    }
+
+    ext {
+        data_prepper_jar_filepath = "build/bin/${DATA_PREPPER_CORE_JAR.archiveName}"
```

Comment:
We use Groovy for Gradle build files and should follow the correct coding standards. This should be camelCase: `dataPrepperJarFilepath`.

Also, this value should be scoped correctly. I'm guessing this should be the root project. In which case it should look more like `"${project.rootProject.buildDir}/bin/${DATA_PREPPER_CORE_JAR.archiveName}"`.

Reply:
Actually it should be the scope of qa-test project, I will replace it with project(':qa-test').buildDir

Reply:
Instead of that, please use `${project.buildDir}` which will use the current project.